geode-docs/tools_modules/redis_api_for_geode.html.md.erb

```diff
@@ -91,8 +90,8 @@ exactly as expected.
 -   **Strings**: BITCOUNT, BITOP, BITPOS, DECRBY, GETBIT, GETSET, MSET, MSETNX, PSETEX, SETBIT, SETEX, SETRANGE
 
 
-If you already have some Geode servers running with Redis enabled, you can execute the following
-command with gfsh to enable unsupported commands:
+If you already have some Geode servers running with Redis Compatibility enabled, you can execute the
+following command with gfsh to enable unsupported commands:
```

Comment:
If you already have <%=vars.produce_name%>  servers enabled with the APIs compatible with Redis, you can execute the

@ringles & @nonbinaryprogrammer  Should we remove this content as well?